Foundation/NSData.swift

```diff
@@ -887,7 +935,17 @@ open class NSData : NSObject, NSCopying, NSMutableCopying, NSSecureCoding {
         }
         return result
     }
-    
+
+    // MARK: Other methods
+    /// Returns a Boolean value indicating whether the sequence contains the given element.
+    func contains(_ element: UInt8) -> Bool {
+        NSUnimplemented()
+    }
```

Comment:
Where did contains and contains(where) come from? As far as I know NSData doesn't conform to Sequence (struct Data does).

Reply:
I based the missing methods on this documentation:

https://developer.apple.com/documentation/foundation/nsdata

Which has the 2 contains methods. Maybe the documentation is incorrect?

Reply:
I don't see them in the generated interface for `NSData` on Darwin. I'll file a bug for the documentation.